testing/test_lib/integration_base.py

```diff
@@ -28,7 +29,7 @@ def _run_forch_script(self, script, arglist=[]):
         """Runs scripts from forch base folder"""
         path = os.path.dirname(os.path.abspath(__file__)) + '/../../'
         command = path + script
-        command = ['sudo', command] + arglist
+        command = [command] + arglist
```

Comment:
same variable (command) should not be assigned twice, so either inline or use an intermediate variable name

Reply:
Done